server/container/core/src/main/java/org/apache/james/server/core/MimeMessageUtil.java

```diff
@@ -75,10 +75,6 @@ public static void writeTo(MimeMessage message, OutputStream headerOs, OutputStr
      */
     public static void writeTo(MimeMessage message, OutputStream headerOs, OutputStream bodyOs, String[] ignoreList) throws IOException, MessagingException {
         MimeMessage testMessage = message;
```

Comment:
this variable is no longer necessary, you can use message directlyon line 78 and 80 (because of the return) and as previously is the special casing for MimeMessageWrapper really useful ?

Reply:
It do it as it allows to get an InputStream. javax.mail enforces a copy by writing to an output stream.

Reply:
ok, thanks for dropping the unnecessary variable